packages/expect/src/__tests__/matchers.test.js

```diff
@@ -247,12 +247,15 @@ describe('.toEqual()', () => {
     [new Map(), new Set()],
     [new Set([1, 2]), new Set()],
     [new Set([1, 2]), new Set([1, 2, 3])],
+    [new Set([[1], [2]]), new Set([[1], [2], [3]])],
+    [new Set([[1], [2]]), new Set([[1], [2], [2]])],
```

Comment:
It might be worth adding a Set of a Set in these tests i.e.

```js
    [
      new Set([new Set([1]), new Set([2])]),
      new Set([new Set([1]), new Set([3])]),
    ]
```

Reply:
Good idea! Added requested tests for both ``Map`` and ``Set`` for `.toEqual` and `not.toEqual`.